src/assets/js/utils.js

```diff
@@ -143,11 +143,29 @@ class AppStorage {
   }
 
   get annee() {
-    return app.data?.["déclaration"]?.["année_indicateurs"]
+    return this.data["déclaration"]?.["année_indicateurs"]
   }
 
   get siren() {
-    return app.data?.["entreprise"]?.["siren"]
+    return this.data["entreprise"]?.["siren"]
+  }
+
+  get isDraft() {
+    // TODO: replace with the following once the API allows it
+    // return this.data.déclaration.statut === "brouillon"
+    return this.data._statut === "brouillon"
+  }
+
+  set isDraft(value) {
+    // TODO: send to the server when toggling to "brouillon" status
+    this.data._statut = value ? "brouillon" : ""
```

Comment:
Ça va changer, ce sera `this.data.déclaration.brouillon` avec soit la valeur `true`, soit le champ n'existera pas (ou sera false)

Reply:
En l'était je reçois l'erreur `data.déclaration must not contain {'brouillon'} properties`…

Reply:
Il fallait que l'API soit mise à jour, c'est maintenant chose faite